applications/nrf5340_audio/src/audio/audio_datapath.c

```diff
@@ -123,6 +123,11 @@ LOG_MODULE_REGISTER(audio_datapath, CONFIG_LOG_AUDIO_DATAPATH_LEVEL);
 /* How often to print underrun warning */
 #define UNDERRUN_LOG_INTERVAL_BLKS 5000
 
+#define SYNC_LED_PERIOD_US 640000
```

Comment:
Many great changes. Though, we cannot have this define, as we will run into offset issues if we do this. 
E.g. if CONFIG_AUDIO_FRAME_DURATION_US = 7500, SYNC_LED_SEQ_NR_PERIOD = 640000: SYNC_LED_SEQ_NR_PERIOD would be 85.33

I would remove this line, and instead just have:
`#define SYNC_LED_SEQ_NR_PERIOD 64` /* Value must be 2^N*/`